### ISSUE TITLE
OpenShift route object has incorrect line endings in manifest

### ISSUE THREAD

**Issue by manderson-it** [hashicorp/vault-helm#346]
Problem: The resulting file opened with **vim** shows the OpenShift Route object has `^M` line endings. This is not the case for other objects in the generated by that helm chart.
Helm Chart: 0.6.0
Helm command:
```
    helm template vault hashicorp/vault --namespace "xyz"
    --values "values-minimal.yaml"
    --set='server.image.repository=hashicorp/vault-enterprise'
    --set='server.image.tag=1.4.2_ent'
    --set='server.ha.enabled=true'
    --set='server.ha.raft.enabled=true'
    --set='server.route.enabled=true'
    --set='server.extraEnvironmentVars.VAULT_CACERT=/vault/userconfig/vault-tls/vault.ca'
    --set='server.tls.secretName=vault-tls'
    --set='global.tlsDisable=false'
    --set='global.openshift=true'
    > "/home/user/vault-all.yaml"
```
Below is the content of values-minimal.yaml supplying parameters to the above helm template command.
```
server:
  ha:
    raft:
      config: |
        ui = true

        listener "tcp" {
          tls_disable = 0
          tls_cert_file = "/vault/userconfig/vault-tls/vault.crt"
          tls_key_file  = "/vault/userconfig/vault-tls/vault.key"
          address = "[::]:8200"
          cluster_address = "[::]:8201"
        }

        storage "raft" {
          path = "/vault/data"
        }

        service_registration "kubernetes" {}
```
When looking at the resulting file with **vim** the Route object has `^M` line endings. This is not the case for other objects in the generated by that helm chart.
```
---
# Source: vault/templates/server-route.yaml
kind: Route^M
apiVersion: route.openshift.io/v1^M
metadata:^M
  name: vault^M
  labels:^M
    helm.sh/chart: vault-0.6.0^M
    app.kubernetes.io/name: vault^M
    app.kubernetes.io/instance: vault^M
    app.kubernetes.io/managed-by: Helm^M
spec:^M
  host: chart-example.local^M
  to:^M
    kind: Service^M
    name: vault-active^M
    weight: 100^M
  port:^M
    targetPort: 8200^M
  tls:^M
    termination: passthrough
```
Please find the full **vault-all.yaml** attached as a zip archive. **Open with vim**.
[vault-all.yaml.zip](https://github.com/hashicorp/vault-helm/files/4845300/vault-all.yaml.zip)

**Comment by jasonodonnell** [comment]
Hi @manderson-it, thanks for the bug report! Does this cause an error or just a cleanup?  Either way we'll get it fixed!

**Comment by manderson-it** [comment]
@jasonodonnell `oc` and `kubectl` still process it correctly, so strictly speaking not a bug. It is peculiar though, that it's the only object generated with `^<` line endings of all generated objects.

On another note, the OpenShift Route object is currently not namespaced, but could or maybe even should be. See the missing `namespace:` line in my first message above.

**Comment by jasonodonnell** [comment]
Thanks for the additional info @manderson-it, marking this as a bug and we'll clean it up.

**Comment by tvoran** [comment]
It looks like this was fixed in #570, thanks for the report!
